src/components/Message/styles/Message.css.js

```diff
@@ -60,6 +60,11 @@ const css = `
     margin-bottom: 0;
     min-width: initial;
   }
+
```

Comment:
This is an override for when a message only includes an emoji and no other related messages. 

<img width="192" alt="Screen Shot 2020-03-03 at 12 58 58" src="https://user-images.githubusercontent.com/7111256/75809682-c6102300-5d4e-11ea-92eb-c18caff7049f.png">